src/runtime/internal/dev.ts

```diff
@@ -128,7 +128,7 @@ export interface SvelteComponentDev {
 	$destroy(): void;
 	[accessor: string]: any;
 }
-interface IComponentOptions<Props extends Record<string, any> = Record<string, any>> {
+export interface ComponentConstructorParams<Props extends Record<string, any> = Record<string, any>> {
```

Comment:
I think the `Params` would not accurately describe the interface as the constructor only has 1 parameter, perhaps `ComponentConstructorOptions` or just `ComponentOptions` would better represent it?

Reply:
Not sure how this happened, I changed this already and made it public - at least I thought so

Reply:
True